docs/components/best-practices/modeling/choosing-the-resource-binding-type.md

```diff
@@ -63,6 +63,23 @@ Camunda 8 supports the following binding types:
         </ul>
       </td>
     </tr>
+    <tr>
+      <td><code>versionTag</code></td>
+      <td>
+        <p>Resolves to the specific version of the target resource that is annotated with the given <strong>version tag</strong>.</p>
+        <ul>
+          <li><p>The version tag is a user-provided string (for example <code>1.2.0.Final</code>) that makes it easy to identify a certain version of a resource and track it across multiple deployment stages (e.g. dev, test, prod).</p></li>
+          <li><p>Using the <code>versionTag</code> binding option ensures that the right version of the target resource is always used, regardless of future deployments, by pinning the dependency to a specific version.</p></li>
+          <li><p>The option is ideal for managing external or shared dependencies.</p></li>
```

Comment:
This is a good place to mention the drawback of the versionTag binding, and what to do
Something like: If the target resource ID and version tag pair are not deployed, the process instance will have an {WHAT TYPE?} incident. To avoid this situation, ensure the version tag defined in the call activity, user task, or business rule task matches the version tag in the dependent resource.

Reply:
Thanks, good suggestion. I've included such a heads-up in the description.
By the way, I left out the incident type, because it's slightly different for call activities, business rule tasks, and user tasks (and I'd say these technical details are not so important here).